cmd/export_ledger_entry_changes.go

```diff
@@ -101,7 +104,18 @@ be exported.`,
 								cmdLogger.LogError(fmt.Errorf("error transforming account entry last updated at %d: %s", entry.LastModifiedLedgerSeq, err))
 								continue
 							}
-							transformedAccounts = append(transformedAccounts, acc)
+							transformedOutputs["accounts"] = append(transformedOutputs["accounts"], acc)
+							if change.AccountSignersChanged() {
```

Comment:
If the accounts signers change and that is the only change to the account, does the `accounts` data change at all? Or would we get a "changed" record that duplicated the last ledger modified sequence?

Reply:
I see. I added a conditional to only update `accounts` only if `AccountChangedExceptSigners`.